rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -620,74 +619,67 @@ void RSocketStateMachine::handleUnknownStream(
     return;
   }
 
-  switch (frameType) {
-    case FrameType::REQUEST_CHANNEL: {
-      Frame_REQUEST_CHANNEL frame;
-      if (!deserializeFrameOrError(frame, std::move(serializedFrame))) {
-        return;
-      }
-      VLOG(3) << "In: " << frame;
-      auto stateMachine =
-          streamsFactory_.createChannelResponder(frame.requestN_, streamId);
-      auto requestSink = requestResponder_->handleRequestChannelCore(
-          std::move(frame.payload_), streamId, stateMachine);
-      stateMachine->subscribe(requestSink);
-      break;
+  if (!isNewStreamFrame(frameType)) {
+    std::stringstream msg;
+    msg << "Unexpected frame " << frameType << " for stream " << streamId;
+    LOG(ERROR) << msg.str();
```

Comment:
I think this should be a `VLOG` or deleted.  I can see this getting spammy with rogue clients, and it's better to just stay away from that.